app/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "postinstall": "npm rebuild --runtime=electron --target=1.6.6 --disturl=https://atom.io/download/atom-shell --build-from-source",
```

Comment:
@Empact also thoughts on updating `--target` to the electron version we are packaging?

Reply:
Yeah fixed in the current code.